app/com/m3/octoparts/model/config/HttpPartConfig.scala

```diff
@@ -19,10 +21,15 @@ import scala.util.Try
 case class HttpPartConfig(id: Option[Long] = None, // None means that the record is new
                           partId: String,
                           owner: String,
-                          uriToInterpolate: String,
                           description: Option[String],
+                          uriToInterpolate: String,
                           method: HttpMethod.Value,
                           additionalValidStatuses: Set[Int] = Set.empty,
+                          httpPoolSize: Int,
+                          httpConnectionTimeout: FiniteDuration,
+                          httpSocketTimeout: FiniteDuration,
+                          httpDefaultEncoding: Charset,
+                          httpProxy: Option[String] = None,
```

Comment:
Is there any easy way to group these into a single `HttpClientConfig` parameter without fighting to hard w/ Skinny-ORM? There are now 25 constructor args / accessors now and that seems a bit much.